### ISSUE TITLE
Multiple matches found in dependency management error

### ISSUE THREAD

**Issue by email2vimalraj** [joeferner/node-java-maven#8]
My java key in `package.json`:

```
"java": {
    "dependencies": [
        {
            "groupId": "ch.qos.logback",
            "artifactId": "logback-classic",
            "version": "1.1.2"
        }
    ]
}
```

I'm getting the following error:
`Error: multiple matches found in dependency management for ch.qos.logback:logback-core:null`

The pom.xml of the logbock-parent is available in: http://search.maven.org/#artifactdetails%7Cch.qos.logback%7Clogback-parent%7C1.1.3%7Cpom

**Comment by joeferner** [comment]
I just published v0.0.10. Can you try again.

**Comment by email2vimalraj** [comment]
This issue is solved. But I have one another issue which I'll open in another thread.
